test/core/api/service/state/state_api_test.cpp

```diff
@@ -294,4 +302,23 @@ namespace kagome::api {
                          common::UnhexError::NON_HEX_INPUT);
   }
 
+  /**
+   * @given state api
+   * @when call getMetadata
+   * @then we recieve correct data
```

Comment:
```suggestion
   * @then we receive correct data
```
Make this mistake every time myself.

Reply:
fixed